packages/framework/src/Component/Elasticsearch/IndexFacade.php

```diff
@@ -63,6 +64,11 @@ public function create(IndexDefinition $indexDefinition, OutputInterface $output
             $indexDefinition->getDomainId()
         ));
 
+        $existingIndexName = $this->indexRepository->findCurrentIndexNameForAlias($indexDefinition->getIndexAlias());
+        if ($existingIndexName !== $indexDefinition->getVersionedIndexName()) {
+            throw new ElasticsearchAliasUsedByDifferentIndex($indexDefinition->getIndexAlias());
+        }
+
```

Comment:
You should catch `ElasticsearchIndexException` here in case that index does not exists yet, right?

Reply:
Is it still relevant?

Reply:
Yes

Reply:
Ah, I see... I do not need to catch that exception. When index is already there it will not creating one.